network/rtpdump.ksy

```diff
@@ -0,0 +1,73 @@
+meta:
+  id: rtmpdump
+  title: Rtpdump (rtptools)
+  file-extension: rtmpdump
+  license: Unlicense
+  ks-version: 0.1
+  endian: be
+  imports:
+    - /network/rtp_packet
+doc: |
+  rtpdump is a format used by rtptools to record and replay
+  rtp data from network capture.
+doc-ref: https://chromium.googlesource.com/external/webrtc/stable/talk/+/master/media/base/rtpdump.h
+seq:
+  - id: file_header
+    type: header_t
+  - id: packets
+    type: packet_t
+    repeat: eos
+-includes:

```

Comment:
Please don't use the dash-prefixed properties in the official KS format gallery, especially when they just mirror what's already present in `meta/imports`.

Reply:
I was not sure if it was relevant to have both "includes" and "imports".
Remove the dash-prefixed as instructed on another commit.